main_measure.py

```diff
@@ -33,7 +33,7 @@ def main():
         'att_cnt': gender_based.att_cnt,
         'att_dis': gender_based.att_dis,
         'att_clu': gender_based.att_clu,
-        'geo_ctr': geography_based.geo_ctr, 
+        'geo_ctr': geography_based.custom_geo_ctr, 
```

Comment:
change to be just geo_ctr instead of custom_geo_ctr (if the consolidation and such is all done?)

Reply:
changed